lab/src/Microsoft.SyndicationFeed/src/Rss20/Rss20Formatter.cs

```diff
@@ -27,14 +42,39 @@ public string Format(ISyndicationContent content)
                 throw new ArgumentNullException(nameof(content));
             }
 
-            using (XmlWriter writer = CreateXmlWriter(out StringBuilder sb))
+            int start = 0;
+
+            //
+            // Create a wrapper that contains the known attributes
+            if (_wrapperWriter == null)
             {
-                writer.WriteSyndicationContent(content);
+                _wrapperWriter = CreateXmlWriter(out _wrapperBuilder);
 
-                writer.Flush();
+                _wrapperWriter.WriteStartElement("w");
+                
+                //
+                // When writing for the first time, the writer will only write "<w" 
+                // As we clear the whole string, when we write the content we have "><content>"
+                // Set start to 1 to ignore ">".
+                start = 1;
 
-                return sb.ToString();
+                if (_attributes != null)
+                {
+                    foreach (var a in _attributes)
+                    {
+                        _wrapperWriter.WriteSyndicationAttribute(a);
+                    }
+                }
+                _wrapperWriter.Flush();
             }
+
+            _wrapperBuilder.Remove(0, _wrapperBuilder.Length);
```

Comment:
Lets use StringBuilder.Clear and clear after writing so we aren't holding a buffer with the formatted content.